src/OpenTelemetry.Contrib.Instrumentation.Wcf/README.md

```diff
@@ -20,6 +20,33 @@ using var openTelemetry = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+Instrumentation can be configured via options overload for 
+`AddWcfInstrumentation` method:
+
+```csharp
+using var openTelemetry = Sdk.CreateTracerProviderBuilder()
+    .AddWcfInstrumentation(options =>
+    {
+        options.SuppressDownstreamInstrumentation = false;
+
+        // Enable enriching an activity after it is created.
+        options.Enrich = (activity, eventName,  message) =>
+        {            
+            var wcfMessage = message as Message;
```

Comment:
this cast depends on the eventName ,right? If yes, please move it inside if(eventname=='') check.

Reply:
this cast always succeed, ok. Ignore my commend above.